jujugui/static/gui/src/app/components/deployment-flow/sshkey/sshkey.js

```diff
@@ -442,7 +450,8 @@ DeploymentSSHKey.propTypes = {
   cloud: PropTypes.object,
   getGithubSSHKeys: PropTypes.func.isRequired,
   setLaunchpadUsernames: PropTypes.func.isRequired,
-  setSSHKeys: PropTypes.func.isRequired
+  setSSHKeys: PropTypes.func.isRequired,
+  user: PropTypes.func
```

Comment:
Should this be an object like in deployment-flow.js?

Reply:
I don't even remember now. At one point, PropTypes yelled at me that it was expecting a func, not an object. I'll dig into that again.

Reply:
This should be an obj; you're using props.user.displayName above, indicating this is the user class object.

Reply:
That said, you're only using the displayName, and that doesn't change for a user once logged in, so you could redo this to pass in only the displayName and do props.String.

I don't think that actually *gets* you anything except that it might be easier for testing, but :: shrugs :: ?